frontend/src/js/controllers/challengeCtrl.js

```diff
@@ -793,6 +793,66 @@
             utilities.sendRequest(parameters);
         };
 
+        vm.showRemainingSubmissions = function() {
+            var parameters = {};
+            vm.remainingSubmissions = {};
+            vm.remainingTime = {};
+            vm.hours = {};
+            vm.minutes = {};
+            vm.remainingSeconds = {};
+            vm.hoursLeft = {};
+            vm.minutesLeft = {};
```

Comment:
I think this variable will take number so why you are initailizing this with an empty object?